app/services/articles/feed.rb

```diff
@@ -25,7 +25,7 @@ def latest_feed
         page(@page).per(@number_of_articles)
     end
 
-    def default_home_feed_and_featured_story(user_signed_in: false)
+    def default_home_feed_and_featured_story(user_signed_in: false, ranking: true)
```

Comment:
Added another parameter here to turn on and off ranking for our home page experiment